docs/actions-pipelines.md

```diff
@@ -60,7 +56,7 @@ The `run_model` action will run a Stata script called `model.do` based on the `d
 It will output two moderately sensitive files `cox-model.txt` and `survival-plot.png`, which can be checked and released if appropriate.
 
 
-Every `project.yaml` requires a `version`, `expectations`, and `actions` section.
+Every `project.yaml` requires a `version` and `actions` section. An `expectations` section was also required for version 3.0 of the pipeline framework (example [here](case-control-studies.md)), but is not used in the latest version.
```

Comment:
```suggestion
Every `project.yaml` requires a `version` and an `actions` section.
```

I don't think we should reference cohort-extractor at all here; we've removed all references to it from everywhere else (except a historical reference to it in the type 1 opt outs description, and this one example). Although that case controls example uses a cohort-extractor action (and therefore a 3.0 project.yaml), there are no docs on any of the actual code there. Anyone looking at that page for the first time has no reference for anything in it, so I think we should just leave it out of here too.

Related: that case-controls-study example should be updated, but that's not for this PR

Reply:
I've added a comment to #1573

Reply:
Suggestion committed, given that the case control studies page is being worked on.